src/sonic-yang-models/sonic-acl.yang

```diff
@@ -0,0 +1,231 @@
+module sonic-acl {
+	namespace "http://github.com/Azure/sonic-acl";
+	prefix sacl;
```

Comment:
Let`s use some fixed pattern for prefixes, Such as for Sonic models, use s<model>, so always, sacl, sport etc. (or we can just say acl, port, because we know main Yang models is for Sonic).
For Ietf --> ie_yang, ie_inet etc.

Reply:
Sure we can but that needs to be documented first. We cannot communicate that to everyone during PR review.